src/RulesTest/Library/ControlShouldSupportSetInfoWPFTests.cs

```diff
@@ -77,7 +79,7 @@ public void TestControlShouldSupportSetInfoWPFExpectedControlTypes()
         public void TestControlShouldSupportSetInfoWPFExpectedPlatform()
         {
             string[] expectedFrameworks = { FrameworkId.WPF };
-            string[] unexpectedFrameworks = { FrameworkId.DirectUI, FrameworkId.Edge, FrameworkId.InternetExplorer, FrameworkId.Win32, FrameworkId.WinForm, FrameworkId.XAML };
+            var unexpectedFrameworks = Extensions.GetFrameworkIds().Except(new string[] { FrameworkId.WPF });
```

Comment:
Most of the code uses explicit types instead of `var`. Any chance we can maintain that pattern?